app/models/rule.rb

```diff
@@ -39,12 +41,15 @@ def variable_requirement?
     false
   end
 
-  def display_for_variable value
+  def display_for_variable(_value)
     nil
   end
 
   def apply_to_match(match)
   end
+
+  def associated_file_tags(value)
```

Comment:
should this return an empty array by default?
```suggestion
  def associated_file_tags(value)
    []
```

Reply:
Doesn't really matter as the `nil`s get cleaned up at the end -- but this makes the API more consistent, and I think that is a good thing.